streams/src/test/java/org/apache/kafka/streams/kstream/internals/WindowedStreamPartitionerTest.java

```diff
@@ -144,4 +144,31 @@ public void testWindowedDeserializerNoArgConstructors() {
         windowedDeserializer.close();
         windowedDeserializer1.close();
     }
+    
+    @Test
+    public void testWindowedDeserializerWindowSize() {
```

Comment:
This method tests multiple things at once and thus should be split into multiple methods. Also, use self describing names for the methods. For example for the first test: `shouldWrapUserSerdeFromConfig()` or similar.